CRM/Report/Form/Contribute/Summary.php

```diff
@@ -827,6 +811,7 @@ public function alterDisplay(&$rows) {
       }
 
       $entryFound = $this->alterDisplayAddressFields($row, $rows, $rowNum, 'contribute/detail', 'List all contribution(s) for this ') ? TRUE : $entryFound;
+      $entryFound = $this->alterDisplayContactFields($row, $rows, $rowNum, 'contribute/detail') ? TRUE : $entryFound;
```

Comment:
There seems to be a mis-match in the arguments that function 'alterDisplayContactFields' is expecting.